[2022-07-28 15:09:39,625] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: PFE_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [queued]>
[2022-07-28 15:09:39,858] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: PFE_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [queued]>
[2022-07-28 15:09:39,865] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 15:09:39,867] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-07-28 15:09:39,875] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 15:09:40,207] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): create_historic_table> on 2022-07-28 14:30:00+00:00
[2022-07-28 15:09:40,214] {standard_task_runner.py:52} INFO - Started process 808 to run task
[2022-07-28 15:09:40,254] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'PFE_historic_data', 'create_historic_table', 'scheduled__2022-07-28T14:30:00+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/open_options.py', '--cfg-path', '/tmp/tmpwz0ss3qw', '--error-file', '/tmp/tmpss_agwlp']
[2022-07-28 15:09:40,259] {standard_task_runner.py:80} INFO - Job 103: Subtask create_historic_table
[2022-07-28 15:09:40,891] {task_command.py:371} INFO - Running <TaskInstance: PFE_historic_data.create_historic_table scheduled__2022-07-28T14:30:00+00:00 [running]> on host a2299eb83309
[2022-07-28 15:09:42,037] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=vardhan
AIRFLOW_CTX_DAG_ID=PFE_historic_data
AIRFLOW_CTX_TASK_ID=create_historic_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T14:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-28T14:30:00+00:00
[2022-07-28 15:09:42,117] {base.py:68} INFO - Using connection ID 'postgres_optionsdata' for task execution.
[2022-07-28 15:09:42,154] {dbapi.py:231} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS historic__PFE_ (
            datetime BIGINT,
            high DOUBLE PRECISION,
            low DOUBLE PRECISION,
            open DOUBLE PRECISION,
            close DOUBLE PRECISION,
            volume INTEGER,
            symbol VARCHAR(32) NOT NULL,
            PRIMARY KEY (symbol, datetime)
        )
    , parameters: None
[2022-07-28 15:09:42,193] {python.py:173} INFO - Done. Returned value was: None
[2022-07-28 15:09:42,360] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=PFE_historic_data, task_id=create_historic_table, execution_date=20220728T143000, start_date=20220728T150939, end_date=20220728T150942
[2022-07-28 15:09:42,423] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 15:09:43,219] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
